frb_codegen/src/config.rs

```diff
@@ -78,9 +79,7 @@ pub struct Opts {
     pub manifest_path: String,
     pub dart_root: Option<String>,
     pub build_runner: bool,
-
-    // due to conflict of dart api of multiple blocks, keep it temporarily
-    pub exclude_sync_execution_mode_utility: bool,
+    pub block_index: usize, //NOTE: the index is counted from 1, not 0
```

Comment:
In addition, maybe we should count starting from zero (not one), which is more intuitive to programmers?

Reply:
Now the final code is always counting from zero, even for the Api suffix shown for user. [119e29b](https://github.com/fzyzcjy/flutter_rust_bridge/pull/527/commits/119e29bd5203c2cf7fd92dfc103bf13184c66c26)